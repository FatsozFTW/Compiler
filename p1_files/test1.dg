""
"&!88"
"use \n to denote a newline character"
"very bad string \
abc
"use \" to  for a quote and \\ for a backslash"
                 A  asdf
"unterminated
"also unterminated \"
"backslash followed by space: \ is not allowed"
"bad escaped character: \a AND not terminated