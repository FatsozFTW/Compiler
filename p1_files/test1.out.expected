PLUS [1,1]
MINUS [1,2]
EOF [2,1]
